k8s/charts/rtsp-camera-simulator/values.yaml

```diff
@@ -0,0 +1,7 @@
+image:
+  repository: "devops-repo.isus.emc.com:8116/nautilus/gstreamer:pravega-dev"
+  pullPolicy: "Always"
+
+camera:
+  user: "admin"
+  password: "password"
```

Comment:
Add a CR to the end of the file. There are others missing CR also.

Reply:
added